Lib/defcon/objects/font.py

```diff
@@ -884,6 +893,8 @@ def _get_identifiers(self):
     # object as dirty.
 
     def _get_guidelines(self):
+        if self._info is None:
+            self.info
```

Comment:
I disagree that the getter has obscure side effects, it's a python property with lazy-loading. It's a standard thing to do and I don't see what's bad about it.